src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -180,14 +182,18 @@ public void NonAppJsonTypeServiceDefinitionForConsumes()
         [Fact]
         public void NonHttpsServiceDefinitionForScheme()
         {
-            var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "Swagger", "Validation", "non-https-service-def-scheme.json"));
+            var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "Swagger", "Validation", "non-https-service-def-scheme.json"),
```

Comment:
@olydis did you suggest testing using an md file here, I don't think that would be possible since these tests run directly with the ServiceDefinition (which is the post autorest-core processing step), the tests won't be able to work with a md file directly.
Am I missing something here.